documentation/modules/metrics/proc-jmx-exporter-metrics-kafka-deploy-options.adoc

```diff
@@ -25,7 +25,7 @@ Set the `enableMetrics` property to `true` in order to expose metrics for these
 * OAuth 2.0 authentication and authorization framework
 * Open Policy Agent (OPA) for authorization
 
-To deploy Prometheus metrics configuration in Strimzi, you can use your own configuration or the xref:ref-metrics-prometheus-metrics-config-{context}[example custom resource configuration files] provided with Strimzi:
+To deploy Prometheus metrics configuration in Strimzi, you can use your own configuration or the xref:ref-jmx-metrics-prometheus-metrics-config-{context}[example custom resource configuration files] provided with Strimzi:
```

Comment:
I did quite a bit of rework on this file in https://github.com/strimzi/strimzi-kafka-operator/pull/11307
I also worded this sentence slightly differently.